src/core/server/models/story/index.ts

```diff
@@ -455,3 +465,16 @@ async function retrieveConnection(
   // Return a connection.
   return resolveConnection(query, input, story => story.createdAt);
 }
+
+export async function compileActiveStories(mongo: Db, tenantID: string) {
+  const stories = await collections
+    .stories(mongo)
```

Comment:
You should instead use the already available `collection` that is an alias of `collections.stories`.